src/routes/SlideRoute.ts

```diff
@@ -3,6 +3,8 @@ import SlideController from '../controllers/SlideController'
 
 const router = express.Router()
 
+router.delete('/:id', SlideController.deleteSlide)
```

Comment:
Make sure to authorize this process to users with access token and are owners of the slide. 
You'll need to use the 'authMiddlewear.ts'
```js
router.delete('/:id', authMiddlewear, SlideController.deleteSlide)
```
After using it you'll need to access the `id` from the access token, you'll find it in the `res.local`. Then find the id of the user who added the slides. If they are the same you can perform deletion.